### PR TITLE
FHIR-46802 Rename patient access actor

### PR DESCRIPTION
**Pull request by johnscarter**
Fixes [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802).

Renamed AU eRequesting Patient Access actor to AU eRequesting Patient per FHIR-46802. Updated the description of the actor.

### CODE REVIEW

File: input/pagecontent/actors.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The AU eRequesting Placer is a system that is responsible for initiating and cre
 
 The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process.
 
-- [AU eRequesting Patient Access actor](ActorDefinition-au-erequesting-actor-patientaccess.html)
+- [AU eRequesting Patient actor](ActorDefinition-au-erequesting-actor-patient.html)
 
 The AU eRequesting Patient Access is a system that is the digital interface that allows patients or their representatives to view and manage requests for diagnostic services. It provides access to request information, enabling patients to choose service providers and track the status of requests.
```
Comment (dbojicic) [inline input/pagecontent/actors.md:13]: Please update this text too

Comment (dbojicic) [reply input/pagecontent/actors.md:13]: Commenting here to get an inline comment. The actors are also referenced on the home page; please update name and description there as well.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -8,3 +8,8 @@ This change log documents the significant updates and resolutions implemented fr
 #### Changes in this version
 - New profiles: 
   - AU eRequesting Task ([FHIR-46947](https://jira.hl7.org/browse/FHIR-46947), [FHIR-46950](https://jira.hl7.org/browse/FHIR-46950), [FHIR-46956](https://jira.hl7.org/browse/FHIR-46956), [FHIR-46958](https://jira.hl7.org/browse/FHIR-46958), [FHIR-46959](https://jira.hl7.org/browse/FHIR-46959), [FHIR-47081](https://jira.hl7.org/browse/FHIR-47081), [FHIR-47082](https://jira.hl7.org/browse/FHIR-47082), [FHIR-47083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
+
+- AU eRequesting ServiceRequest
+  - Made AUeRequestingServiceRequest.encounter mandatory per [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
+
+- Renamed AU eRequesting Patient Access actor to AU eRequesting Patient and updated the description text per [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:15]: Don't think we need to mention descriptions, but we should mention CapabilityStatement changes; change to something like:
Renamed AU eRequesting Patient Access actor and CapabilityStatement to be AU eRequesting Patient [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)"

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments

File: input/pagecontent/actors.md [hunk 3]
```diff
@@ -8,9 +8,9 @@ The AU eRequesting Placer is a system that is responsible for initiating and cre
 
 The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process.
 
-- [AU eRequesting Patient Access actor](ActorDefinition-au-erequesting-actor-patientaccess.html)
+- [AU eRequesting Patient actor](ActorDefinition-au-erequesting-actor-patient.html)
 
-The AU eRequesting Patient Access is a system that is the digital interface that allows patients or their representatives to view and manage requests for diagnostic services. It provides access to request information, enabling patients to choose service providers and track the status of requests.
+A system that allows patients or their representatives to view diagnostic requests and fulfilment of diagnostic requests.
```
Comment (dbojicic) [inline input/pagecontent/actors.md:13]: This starts with 'A system..." which is inconsistent with the structure used for other actors. To maintain consistency, please update to: 
>The AU eRequesting Patient is a system that allows patients or their representatives to view diagnostic service requests and track the fulfilment of these requests.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: This renders as a bullet point under AU eRequesting ServiceRequest changes. Please update the formatting to ensure it appears as a first-level bullet point.

File: input/pagecontent/actors.md (hunk 3 above)
Comment (johnscarter) [reply input/pagecontent/actors.md:13]: I used the exact language in the ticket. I'll change it as you point out.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: This is still unresolved.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Just one more minor change

File: input/pagecontent/actors.md (hunk 3 above)
Comment (dbojicic) [reply input/pagecontent/actors.md]: Reviewed changes

**Review by dbojicic — APPROVED** [review]

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**